solutions/cses-1621.mdx

```diff
@@ -1,102 +0,0 @@
----
```

Comment:
hmm, i think this was intentional to keep only the solution with sets in the module

Reply:
but isn't the module Introduction to Sets & Maps? 🤔 should i ask @thecodingwizard?

Reply:
yeah, i'm not too sure about this

what i meant was that the solution using sets would be in the module, since the module is intro to sets and maps, and the solution using sorting would be an internal solution since it doesn't relate to the module

Reply:
I agree w/ Maggie

Reply:
👍

Reply:
i thought the other implementation was maps, sorry about that :)